ch04packaging/03Packaging.ipynb.py

```diff
@@ -362,21 +404,18 @@ def process():
 [project.scripts]
 greet = "greetings.command:process"
 
-[build-system]
-requires = ["setuptools", "setuptools_scm[toml]>=6.2", "wheel"]
-
-[tool.setuptools.packages.find]
-include = ["greetings*"]
+[tool.hatch.build.targets.sdist]
+include = [
+  "greetings/",
```

Comment:
```suggestion
  "src/",
```
that's at least what we have above. `greetings` or would it be `src/greetings/`?

Reply:
Missed that, thanks!